tracks/ruby/exercises/series/mentoring.md

```diff
@@ -0,0 +1,49 @@
+New concepts: Iterating over an array (preferably with `each_cons`, avoid `each_with_index`); `.chars` to split a string; (private) `attr_reader`; raising an Argument Error; inline if-statement 
+
+### Minimal solution for approval
+
+```ruby
+
+class Series
+
+  def initialize(serie)
+    @serie = serie
+  end
+
+  def slices(span)
+    raise ArgumentError if span > serie.size
```

Comment:
Consider using `unless` and showing what "good" looks like for the conditional statement.